tasks/os-tunning.yml

```diff
@@ -15,13 +15,15 @@
     state: present
   with_items: "{{ solr_sysctl_properties }}"
   when: solr_sysctl_properties is defined
+  notify: reboot machine
```

Comment:
Kernel parameter changing via sysctl doesn't require rebooting machine.

Reply:
These changes worked for Debian < 8 but because systemd evolution, in the latest releases doesn't work. They must be changed at systemd unit .service file level.